### ISSUE TITLE
Downloading Features 

### ISSUE THREAD

**Issue by D024kkk** [vega-org/vega-app#105]
### Describe your suggested feature

Please🙏 improve download feature when I download a video and if wifi get disconnected for 1 sec then download failed even if it is at 95% , **Please add pause/resume button**
And downloading should be automatically paused if internet disconnect and can be resumed later

### Acknowledgements

- [x] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by Zenda-Cross** [comment]
Certainly, these features are on the roadmap and will be added if the project continues.

**Comment by D024kkk** [comment]
Ok thank you

**Comment by Zenda-Cross** [comment]
I'll try to add it as soon as I can

**Comment by Aladdin4u** [comment]
The download feature is not working - I can't resume the download once it fails.
